From ee99a5a4a503b5b1b6b6c20876fafafa4dd6b96a Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Fri, 29 Mar 2019 08:24:05 -0400
Subject: [PATCH] Revert "Add a upper limit to sqlalchemy as 1.3.0 breaks our
 tests"

This is required so that Pagure is installable with SQLAlchemy 1.3.0+.
The issues that affect the tests are limited to the SQLite backend,
and do not affect production deployments using PostgreSQL or MySQL/MariaDB.

This reverts commit b92a6f378bc81890ed2c1c70c84bce687238b617.
---
 requirements.txt | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index c877d94e..996fb063 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -26,12 +26,7 @@ python-openid-teams
 redis
 requests
 six
-# sqlalchemy minimum 0.8
-# sqlalchemy 1.3.0 is causing issues on the pip container leading
-# test_pagure_lib.py to raise a:
-# "(sqlite3.OperationalError) no such column: users.user"
-# in test_search_projects_private line 319
-sqlalchemy < 1.3.0
+sqlalchemy >= 0.8
 # 1.4.0 is broken, 1.4.0-post-1 works but gives odd results on newer setuptools
 # the latest version 1.5.0 is also known to work
 straight.plugin
-- 
2.20.1

